source/documentation/get_data/api_documentation.md

```diff
@@ -14,29 +14,21 @@ For example calls using curl and parameters see the [CKAN API documentation](htt
 
 This table lists the parameters and returns with example URLs.
 
-<div style="height:1px;font-size:1px;">&nbsp;</div>
-
-| Parameter                         | Returns                            | Example URL                                                              |
-|-----------------------------------|------------------------------------|--------------------------------------------------------------------------|
-| `package_list`                    | List of datasets                   | https://data.gov.uk/api/action/package_list                              |
-| `package_show?id=<PUBLISHER-NAME>` | Information about a single dataset | https://data.gov.uk/api/action/package_show?id=cabinet-office-energy-use |
-
-<div style="height:1px;font-size:1px;">&nbsp;</div>
```

Comment:
Wait, really?? https://github.com/alphagov/tdt-documentation/blame/master/source/write_docs/content/index.html.md.erb#L25

Reply:
I'm chasing this up in the tech writers community to find out the reason behind it and whether it can be removed.

Reply:
Looks like a legacy requirement as tables used to not render unless we added this hack. Checking this locally, the tables render fine, although the spacing between tables and content is reduced without the `<div>`. That's something that should be fixed up by the central tech docs gem CSS if it is deemed an issue.

These `<div>`'s are safe to remove.

Reply:
Have raised a PR to update the guidance: https://github.com/alphagov/tdt-documentation/pull/149